patches/chrome-browser-resources-password_manager-password_manager.ts.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/password_manager/password_manager.ts b/chrome/browser/resources/password_manager/password_manager.ts
+index 00636b8e4f0aabe012d5c4cd4d18c55b7962334c..df50ff13942549c891906f6b04e0fe4003a06193 100644
+--- a/chrome/browser/resources/password_manager/password_manager.ts
++++ b/chrome/browser/resources/password_manager/password_manager.ts
+@@ -2,6 +2,8 @@
+ // Use of this source code is governed by a BSD-style license that can be
+ // found in the LICENSE file.
+ 
++import 'chrome://resources/brave/polymer_overriding.js'
```

Comment:
why do we need this here if we include it in `brave_overrides/index.js`?

Reply:
Fixed in https://github.com/brave/brave-core/pull/19398/commits/8359b1f81a35867e35dbe2f7803f8ffef0f9749c